### ISSUE TITLE
Chia-anime tv feature request

### ISSUE THREAD

**Issue by rawdpiper** [Xonshiz/anime-dl#33]
I want to request for a script/windows binary that can download anime from chia-anime.tv which we'll provide the URLs to which the anime episodes are listed
The Features that I'll love to be present in the script are:
1. Searching for a particular anime instead of having to open the browser to get the link
2. Select the video quality i.e 1080p, 720p, 480p or 360p
3. Storing the download links to a .txt file so that they can be added through batch download on IDM
4. Episode Range if one chooses not to download the whole episodes

**Comment by rawdpiper** [comment]
Hello @Xonshiz, any update on this?

**Comment by Xonshiz** [comment]
No. I'm super busy this whole month and cannot look into anything else. I still have comic-dl to tend to and then this. It'll take time, sorry.
